playwright.config.ts

```diff
@@ -7,7 +7,7 @@ const config: PlaywrightTestConfig = {
   testDir: './test',
   use: {
     trace: 'on-first-retry',
-    baseURL: process.env.CI ? process.env.BASE_URL : 'http://localhost:3000',
+    baseURL: 'http://localhost:3000',
```

Comment:
Ideally, we'd test against the vercel deployment, but I think for now this is better than nothing?